### PR TITLE
Feat/systemwide metrics

### PR DESCRIPTION
**Pull request by jgryffindor**
Adding systemwide metrics endpoints for. This is useful for specifying custom queries that aren't related directly to a named Prometheus query but run against the local database. 

## Description

- Cleaned up client queries, removed unused parameters
- Added systemwide controller, module, service
- Updated metricChart and metricStat components with prop to trigger systemwide endpoint for metric by name
- Added tokens.create method details and details page to transactions page
- Added tokens.create to PrettyMethods 
- Adding responsive breakpoints to stack components on home page and network metrics page on mobile

## Related Issue

Fixes # (issue)

## Testing

Local tests

## Breaking Changes (if applicable)
None 

## Screenshots
System-wide metrics: 
image

tokens.create Transaction Details: 
image

## Checklist:

- [x] I have read and followed the CONTRIBUTING guidelines for this project.
- [ ] I have added or updated tests and they pass.
- [ ] I have added or updated documentation and it is accurate.
- [x] I have noted any breaking changes in this module or downstream modules.

### CODE REVIEW

File: client/src/pages/transaction/index.ts [hunk 1]
```diff
@@ -5,3 +5,4 @@ export * from "./ledger-send";
 export * from "./multisig";
 export * from "./tokens-mint-burn";
 export * from "./transaction";
+export * from "./tokens-create";
```
Comment (fmorency) [inline client/src/pages/transaction/index.ts:8]: EOF EOL

File: client/src/pages/transaction/tokens-create.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+export function tokensCreate(data: any) {
+  const summary = data.argument.summary;
+  return {
+    Token: (
+      <Code as={Link} to={`/addresses/${data.argument.owner}`} color="brand.teal.500">
+        {summary.ticker}
+      </Code>
+    ),
+    Holders: (
```
Comment (fmorency) [inline client/src/pages/transaction/tokens-create.tsx:12]:
```suggestion
    Initial Distribution: (
```

File: client/src/pages/transaction/tokens-create.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      <Table>
```
Comment (fmorency) [inline client/src/pages/transaction/tokens-create.tsx:13]: How does it render when you have more than a single dest?

File: client/src/ui/metricStat.tsx [hunk 4]
```diff
@@ -46,7 +53,13 @@ export function MetricStat({
...
 
   if (!isLoading && queryData) {
-    metricValues?.push(Number(queryData.data));
+
+    // Apply type to destructured queryData
+   let statData: QueryData = queryData;
+
+    metricValues?.push(Number(statData.data));
+
+    console.log(`MetricStat Data ${label}: ${statData.data}`)
```
Comment (fmorency) [inline client/src/ui/metricStat.tsx:62]: Remove

File: server/src/metrics/systemwide/systemwide.service.ts [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    if (!values) {
+      return null;
+    }
+
+    const blocksum = values.reduce(
+      (accumulator, item) => accumulator + Number(item.highest_value),
+      0,
+    );
+
+    this.logger.debug(`Total blocks: ${blocksum}`);
```
Comment (fmorency) [inline server/src/metrics/systemwide/systemwide.service.ts:53]: Remove?

File: server/src/metrics/systemwide/systemwide.service.ts [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+  // Get the total transactions from all networks
+  async getTotalTransactions(): Promise<Current> {
+    const query = this.transactionRepository
+      .createQueryBuilder("t")
+      .select("COUNT(t.id)", "count");
+
+    const transactions = await query.getRawOne();
+
+    this.logger.debug(`Total transactions: ${transactions.count}`);
```
Comment (fmorency) [inline server/src/metrics/systemwide/systemwide.service.ts:72]: Remove?

File: client/src/pages/transaction/tokens-create.tsx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+              </Td>
+              <Td>
+                <Code
+                  as={Link}
+                  to={`/addresses/${address}`}
+                  color="brand.teal.500"
+                >
+                  {address}
+                </Code>
+              </Td>
```
Comment (fmorency) [inline client/src/pages/transaction/tokens-create.tsx:32]: I would swap the two columns, i.e., `address: amount TOKEN`

**Review by fmorency — CHANGES_REQUESTED** [review]
Minor changes! Thanks!

File: client/src/pages/transaction/tokens-create.tsx (hunk 3 above)
Comment (jgryffindor) [reply client/src/pages/transaction/tokens-create.tsx:13]: It renders a table of each holder with the initial distribution. Would you prefer a list of holders and a single distribution value?

Comment (fmorency) [reply client/src/pages/transaction/tokens-create.tsx:13]: Is it easy to show me both?

File: client/src/pages/transaction/tokens-create.tsx (hunk 7 above)
Comment (jgryffindor) [reply client/src/pages/transaction/tokens-create.tsx:32]: image

File: client/src/pages/transaction/tokens-create.tsx (hunk 3 above)
Comment (jgryffindor) [reply client/src/pages/transaction/tokens-create.tsx:13]: Here's the data I'm receiving. Are there cases where the initial distribution will differ per address? 
```
{
  "summary":{
     "name":"Test",
     "ticker":"TST",
     "precision":9
  },
  "owner":"mahmcjf5yyodcpomqrcnqfkbpltt45oygniar7y32p7ltykqnd",
  "holders":{
     "mahmcjf5yyodcpomqrcnqfkbpltt45oygniar7y32p7ltykqnd":"10000000000000000"
  },
  "extendedInfo":{
     
  },
  "memo":[
     
  ]
}
```

Comment (fmorency) [reply client/src/pages/transaction/tokens-create.tsx:13]: Yes. The initial distribution can be anything. E.g.,
```json
{
  "maaaaaaaaaaaaaaaaaaaaaaaaaaaaaa": 123,
  "mabbbbbbbbbbbbbbbbbbbbbbbbbbbbb": 456,
}
```

File: client/src/pages/transaction/tokens-create.tsx (hunk 7 above)
Comment (fmorency) [reply client/src/pages/transaction/tokens-create.tsx]: Not a fan of the additional horizontal bar.

File: client/src/pages/transaction/tokens-create.tsx (hunk 3 above)
Comment (jgryffindor) [reply client/src/pages/transaction/tokens-create.tsx:13]: Then I suggest we leave it as the table

File: client/src/pages/transaction/tokens-create.tsx (hunk 7 above)
Comment (jgryffindor) [reply client/src/pages/transaction/tokens-create.tsx]: Okay, fixed on both create token table and mint-burn tables 
image

**Comment by fmorency** [comment]
Can you fix the conflict please?

**Review by fmorency — APPROVED** [review]
LGTM assuming conflict is fixed.

**Pull request merged**